tests/main/revert/task.yaml

```diff
@@ -1,5 +1,11 @@
 summary: Check that revert works.
 
+details: |
+    Check that when a snap is refresh it can be reverted to the previous
+    version. Verify that the data directories are present and the snap
+    runs confined. Also check that a revert on an already reverted snap
+    should fail
```

Comment:
```suggestion
    Check that when a snap is refreshed it can be reverted to the previous
    version. Verify that the data directories are present and the snap
    runs confined. Also check that a revert on an already reverted snap
    fails
```